app/client/components/accountoverview/contributionUpdateAmountForm.tsx

```diff
@@ -14,7 +14,7 @@ import { trackEvent } from "../analytics";
 import AsyncLoader from "../asyncLoader";
 import { Button } from "../buttons";
 import { SuccessMessage } from "../delivery/address/deliveryAddressEditConfirmation";
-import { Input } from "../delivery/address/input";
+import { Input } from "../input";
```

Comment:
Is it possible to give it more meaningful name than than `input`?

Reply:
This is for an input component and follows design systems naming convention